test/strong_migrations_test.rb

```diff
@@ -368,8 +374,13 @@ def test_down
     assert_safe SafeUp, direction: :down
   end
 
+  def test_add_foreign_key
+    skip unless postgres?
```

Comment:
It has been a really long time since I've used mysql. How confident are we that the unsafe foreign key does not apply to it?

Reply:
I'm not sure if it's safe with MySQL, it's fine to start with just Postgres.